convert/convert.go

```diff
@@ -28,10 +29,22 @@ func run(path string, overwrite bool) error {
 	if err != nil {
 		return err
 	}
+
+	// Look for a .gunkconfig
+	absPath, _ := filepath.Abs(path)
+	cfg, err := config.Load(filepath.Dir(absPath))
+	if err != nil && err != config.ErrNoConfig {
+		return fmt.Errorf("unable to load .gunkconfig: %v", err)
+	}
+	var convertor *config.Convertor
+	if cfg != nil && cfg.Convertor != nil {
+		convertor = cfg.Convertor
+	}
```

Comment:
I think this can be much simpler:

```
var t T
cfg, err := config.Load("")
if err == nil {
    t = cfg.T
}
```

I don't think differentiating config load errors is necessary. If `convert` doesn't need a config file, I don't see why it should error if an invalid config file is on disk.